roles/st2repos/tasks/redhat.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Install RPM key for StackStorm
+  become: yes
+  rpm_key:
+    state: present
+    key: https://packagecloud.io/StackStorm/stable/gpgkey
+
+- name: Add ST2 Repo
+  become: yes
+  yum_repository:
+    name: stackstorm
+    description: Stackstorm Repo
+    file: st2
+    baseurl: https://packagecloud.io/StackStorm/stable/el/$releasever/$basearch
```

Comment:
Missing `{{ st2_pkg_repo }}` variable here which is responsible for repository we install (`stable`/`unstable` etc).

See: https://github.com/StackStorm/ansible-st2#variables and existing rule in `deb`: https://github.com/StackStorm/ansible-st2/pull/66/files#diff-37045cfaa2d3524070e708be840837adR21